rllib/algorithms/algorithm_config.py

```diff
@@ -450,8 +450,8 @@ def framework(
 
     def environment(
         self,
-        *,
         env: Optional[Union[str, EnvType]] = None,
+        *,
```

Comment:
This seems to be the better choice:

Simply:
```
config.environment("CartPole-v0")
```

instead of 
```
config.environment(env="CartPole-v0")
```